src/main/java/org/example/domain/Program.java

```diff
@@ -0,0 +1,30 @@
+package org.example.domain;
+
+import java.util.List;
+
+public class Program implements IProgram {
+    private IPersistenceHandler persistenceHandler;
+
+
+    public Boolean createProgram(ProgramEntity programEntity){
+        return false;
+    }
+    public Boolean deleteProgram(ProgramEntity programentity){
+        return false;
+    }
+    public Boolean updateporgam(ProgramEntity programEntity){
+        return false;
+    }
+    public List<ProgramEntity> getAllPrograms(){
+        return null;
+    }
+    public ProgramEntity getProgramByld(String id){
+        return null;
+    }
+    public List<Program> getProgramsByCompany(Company company){
+        return null;
+    }
```

Comment:
CompanyEntity class istedet for Company.

Reply:
done